packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActionButtons/index.js

```diff
@@ -71,14 +79,68 @@ const confirmDialogsPropTypes = {
 
 const ConfirmDialogPublishAll = ({ isOpen, onToggleDialog, isConfirmButtonLoading, onConfirm }) => {
   const { formatMessage } = useIntl();
+  const { get } = useFetchClient();
+  const { selectedEntries } = useTableContext();
+  const toggleNotification = useNotification();
+  const { formatAPIError } = useAPIErrorHandler(getTrad);
+  const {
+    contentType: { uid: slug },
+  } = useSelector(listViewDomain());
+
+  const {
+    data: countDraftRelations,
+    isLoading,
+    isError,
+    error,
+  } = useQuery(
+    ['content-manager', 'draft-relations'],
```

Comment:
@joshuaellis made me aware: you are missing all dynamic dependencies (such as slug).

Reply:
ok, thanks I added the slug and selectedEntries as dependencies